client/src/util/helpers.ts

```diff
@@ -26,6 +28,43 @@ const hpdNumberTransformations = [
   ["TENTH", "10"],
 ];
 
+const hpdComplaintTypeTranslations = new Map([
```

Comment:
This feels iffy to me, but having read https://github.com/lingui/js-lingui/issues/426, I guess it's not an uncommon problem.

Eventually we might want to think about storing these in a database instead of in code kind of like in the issue above. Is this the first time we've ever had to translate raw data? It feels like this is setting a precedent and it would be great to have just a smidge longer eng-specific chat on how we want to handle this in the future.

Reply:
Oh nice good find on that issue from lingui! And that totally makes sense, let's flag this as something to discuss next week at engineering check in.

Reply:
Hmm, I am intrigued by the possibility of doing this translation on the server-side rather than the client-side.

In the tenant platform, our APIs are actually prefixed by a locale identifier (e.g. `/en/graphql` or `/es/graphql`) which allows the API response to be localized.  However, in WoW, it's not prefixed by a locale (it's just `/api/blah`), which makes it harder to do the localization on the server-side.  Still, we could potentially migrate to that kind of URL structure, or we could have the locale get passed in via a querystring argument...

Anyways, I don't think it's _that_ big a deal because our list of strings to be localized isn't too huge.

Storing the translations in a database is possible but comes with its own headaches--most significantly, since dev/staging/production use entirely different databases, we'd have potentially different behavior from them, and it raises the issue of keeping them "in sync"... we could do fancy things with migrations to make sure they are in sync, but then that's functionally the same thing as just translating them via Crowdin.  I think it might be a bigger deal if this list of categories was constantly changing and we wanted to be able to localize it without having to issue a PR, but since the list appears to be fairly stable, I think translating the strings in Crowdin is probably fine.

Regardless of the outcome though I think this is a great thing to flag @samaratrilling, thanks for doing tha!

Reply:
awesome, thanks for the feedback both of you! Given this analysis, I think it's safe to merge this for now although I will make a separate issue to explore doing the translation on the server-side as mentioned.